app.js

```diff
@@ -251,7 +251,11 @@ const userProfile = () => {
 
             if (parameters?.token) {
                 const response = await validateToken(parameters.token); // Add uid and sign-in flag if token is valid
-                if (response.code === 200) {
+                if (response.code === 202 && response.message === 'Duplicate account') {
```

Comment:
is it necessary to check that the message is equal to a hardcoded string as well? is there another scenario where we would send a 202 code?